InputValidation/ValidationMethods/Validation.FirstCharacter.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+using TotalImage.Properties;
+using TotalImage.Validation;
+
+/// <summary>
+/// Validates the first character of a DOS 8.3 filename (in the case it has been overwritten with 0xE5) (2020/08/01)
+/// </summary>
+namespace TotalImage.Validation.ValidationMethods
+{
+    public class FirstCharacterValidation : IValidation
+    {
+        public object Validate(object Data) 
+        {
+            // Return if not char
+            Data = ValidationCore(Data);
+
+            if (Data == null)
+            {
+                OnFail();
+            }
+            else
+            {
+                return Data;
+            }
+
+            //blah
+            return null; 
+
+        }
+
+        private object ValidationCore(object Data)
```

Comment:
Is there any reason this isn't static? It doesn't access any instance fields, and therefore making this static would mean that calling it comes with a lower overhead and potentially gives the compiler better choices for optimising generated code.